app/controllers/settings/general.js

```diff
@@ -33,7 +33,7 @@ export default Controller.extend(SettingsSaveMixin, {
     }),
 
     iconImageSource: computed('model.icon', function () {
-        return this.get('model.icon') || '';
+        return this.get('model.icon') || '/favicon.ico';
```

Comment:
I will test this now, @acburdine. We build redirects `/favicon.ico` and `/favicon.png` requests, which should actually work.